mkosi/qemu.py

```diff
@@ -445,6 +445,29 @@ def qemu_version(config: MkosiConfig) -> GenericVersion:
     return GenericVersion(run([find_qemu_binary(config), "--version"], stdout=subprocess.PIPE).stdout.split()[3])
 
 
+def apply_runtime_size(image: Path, config: MkosiConfig) -> None:
```

Comment:
Please pass in `config` as the first argument as we do it like that everywhere else

Reply:
Fixed in c181839c.